catalog/ui/search-ui/standard/src/main/webapp/templates/search/searchForm.handlebars

```diff
@@ -208,7 +208,7 @@
 
                         <div class="input-group input-group-sm">
                             <span class="input-group-addon">Radius&nbsp;</span>
```

Comment:
Changed this to select the HTML element by ID instead of name and removed the name attribute from radiusValue.